src/StorageSync/StorageSync/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
```

Comment:
remove line 21 and 22. Don't bump the version in the ChangeLog. It'll be automatically done in release.